lib/os/path.py

```diff
@@ -14,7 +14,8 @@
 
 """"Utilities for manipulating and inspecting OS paths."""
 
-from __go__.path.filepath import Clean as normpath, IsAbs as isabs, Join  # pylint: disable=g-multiple-import,unused-import
+from __go__.path.filepath import Clean as normpath, IsAbs as isabs, Join, Dir as dirname  # pylint: disable=g-multiple-import,unused-import
```

Comment:
I think we need a wrapper for Clean() because it doesn't behave the same way as normpath() when the input is unicode:

CPython:
```
>>> path.normpath(u'a/../b')
u'b'
```

Grumpy:
```
print repr(Clean(u'a/../b'))
'b'
```